pkgs/sketch_pad/lib/main.dart

```diff
@@ -588,11 +617,11 @@ class StatusLineWidget extends StatelessWidget {
               const url = 'https://dart.dev/tools/dartpad/privacy';
               url_launcher.launchUrl(Uri.parse(url));
             },
-            child: const Row(
+            child: Row(
               children: [
-                Text('Privacy notice'),
-                SizedBox(width: denseSpacing),
-                Icon(Icons.launch, size: 16),
+                Text(MessagesLocalizations.of(context)!.privacyNotice),
```

Comment:
When to prefer `MessageLocalizations.of(context)` over `context.messageLocalizations`?